cmd/user-agent/src/pages/chapi/wallet/common/presentationExchange.js

```diff
@@ -210,6 +210,13 @@ function matchManifest(manifest, descriptor) {
 
     let schemas = Array.isArray(descriptor.schema.uri) ? descriptor.schema.uri : [descriptor.schema.uri]
 
+    // yuck
```

Comment:
Can we add a TODO here to remove `yuck` ? :)

Reply:
I wanted to [coallesce the null checks](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and tried adding [this](https://www.npmjs.com/package/@babel/plugin-proposal-optional-chaining) dependency but couldn't for unclear reasons.... I think the TODO a couple of lines above will likely refactor the whole function anyway.

Reply:
Removed just to avoid the extra noise